src/backend/joanie/core/models/courses.py

```diff
@@ -31,6 +31,8 @@
 class CourseState(Mapping):
     """An immutable object to describe a course (resp. course run) state."""
 
+    state_count = 8
```

Comment:
can you use the corresponding constant instead?

Reply:
I changed it so that we don't need a const and the tests are automatically testing all states